admin/jsonConfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "panel",
+  "i18n": true,
+  "items": {
+    "_basicSettings": {
+      "newLine": true,
+      "type": "header",
+      "text": "Please provide your credentials",
+      "sm": 12,
+      "size": 8
+    },
+    "Username": {
+      "newLine": true,
+      "type": "text",
+      "label": "Username",
+      "sm": 4
+    },
+    "Password": {
+      "repeat": false,
+      "visible": false,
+      "sm": 3,
+      "type": "password",
+      "label": "Password"
+    },
+    "intervall": {
+      "sm": 3,
+      "type": "number",
+      "label": "Interval in seconds",
+      "min" : 10,
```

Comment:
in the adapter code 30 seconds is a minimum. We should take the 30 here as well

Reply:
clear, updated